server/src/controllers/questionsController.ts

```diff
@@ -313,3 +314,29 @@ export const createCommentHandler = async (req: Request, res: Response) => {
       .json({ error: "Error Adding Your Comment to the Database" });
   }
 };
+
+export const editQuestionHandler = async (req: Request, res: Response) => {
+  const questionId = Number(req.params.id);
```

Comment:
```suggestion
  const questionId = parseInt(req.params.id);
  logger.info({
    message: "editQuestionHandler questionId",
    value: questionId
  });
```

lets change to using `parseInt`` as you suggested
and we need to add a logger here